Readme.md

```diff
@@ -3,38 +3,48 @@
 
 (If you want to skip the spiel and get straight to business, check out the [docs](./documentation.md).)
 
+
 ### What does it do?
 
-Traffic Cop is a small bit of JavaScript in the `<head>` of a page that decides if a visitor should be shown a variation of the current page. If so, a cookie is set and the user is redirected to the current URL with a developer-specified querystring parameter appended. That's it.
+Traffic Cop is a small bit of JavaScript in the `<head>` of a page that decides if a visitor should participate in a variation of the current page. If so, a cookie is set and one of two things happens:
```

Comment:
While the docs suggest callback experiments should not be in the `<head>` for perf reasons, this first line in the readme kinda suggests the library itself should always be loaded this way, which isn't necessarily true now. Suggest simplifying here and just saying something like `Traffic Cop is a small JavaScript library that decides...`?